README.md

```diff
@@ -66,6 +66,24 @@
 
 ## Installation
 
+### Swift Package
+The [Swift Package Manager](https://swift.org/package-manager/ "") is a tool for automating the distribution of Swift code and is integrated into the swift compiler.
+
+Once you have your Swift package set up, adding Alamofire as a dependency is as easy as adding it to the `dependencies` value of your `Package.swift`.
```

Comment:
Never noticed that this section was copied from Alamofire and it wasn't updated to fit `web3swift`. Haha 😄

Reply:
Fixed.